sql/item_create.cc

```diff
@@ -3229,8 +3229,12 @@ Item*
 Create_func_kdf::create_native(THD *thd, const LEX_CSTRING *name,
                                List<Item> *item_list)
 {
-  uint arg_count= item_list->elements;
   Item *a[5];
+  uint arg_count= 0;
+
+  if (item_list != NULL)
+    arg_count= item_list->elements;
```

Comment:
Now I wonder if there are any/many other SQL functions with the similar mistake of checking `item_list->elements` before checking `item_list != NULL`.

Did you search for any of them, by any chance?

Reply:
I did.